### DISCUSSION TITLE
Requesting whitelisted domain to test merged PR wallet code

### DISCUSSION THREAD

**Discussion by BestWeb3** [rainbow-me/rainbowkit#2217] [Q&A]
Hey team,

I wanted to inform you that I have successfully raised the PR, which seems to have been merged into your codebase. We are now eager to move forward with testing our wallet on your domain.
Could you kindly advise on the best practices we should follow to ensure a smooth and efficient testing process?

**Comment by DanielSinclair** [comment]
The connector is available in [2.1.7](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%402.1.7)
